src/docs/_includes/navigation.html

```diff
@@ -77,4 +87,58 @@
     });
   });
 
+  $searchBox.on('keydown', function(e) {
+      var children = $searchResults.children();
+
+      if (e.keyCode === 40) {  //arrowdown
```

Comment:
what about keeping `keyCode`s as `const`s at the very beginning (no additional comments needed then)?